src/model/ProteinNames.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import NameView from './NameView';
+import idx from 'idx';
+import { InfoList } from 'franklin-sites';
+
+export type ProteinNamesData = {
+  proteinDescription: {
+    recommendedName?: {
+      fullName: {
+        value: string;
+      };
+    };
+    submissionNames?: [
+      {
+        fullName: {
+          value: string;
+        };
+      }
+    ];
+    shortNames?: [
```

Comment:
In `modelData.json`, `shortNames` and `ecNumbers` are properties of `proteinDescription.recommendedName` but the paths in `ProteinNames.tsx` assume they are properties of `proteinDescription` which means they're not being accessed and rendered.